src/cli.js

```diff
@@ -298,12 +300,16 @@ program
         }
       })
 
+      const filterQuery = filter ? buildFilterQuery(keys, operations, values) : undefined
+      const token = creds.get().token || null
```

Comment:
Remove this two lines, we don't have the filter query and the token it's already added in the line `294`